pkgs/by-name/bo/bochs/package.nix

```diff
@@ -143,8 +143,10 @@ stdenv.mkDerivation (finalAttrs: {
       Intel x86 CPU, common I/O devices, and a custom BIOS.
     '';
     license = lib.licenses.lgpl2Plus;
+    mainProgram = "bochs";
     maintainers = with lib.maintainers; [ AndersonTorres ];
     platforms = lib.platforms.unix;
+    broken = stdenv.isDarwin;
```

Comment:
The package isn't broken on aarch64-darwin, however? https://hydra.nixos.org/job/nixpkgs/trunk/bochs.aarch64-darwin

Reply:
Still broken on Darwin ARM

Reply:
it's certainly still broken on aarch64-darwin for me
[z1xcamxmlhndf8vrzsk2gknrm4whqdvl-bochs-2.8.drv.log](https://github.com/NixOS/nixpkgs/files/15288092/z1xcamxmlhndf8vrzsk2gknrm4whqdvl-bochs-2.8.drv.log)

Reply:
OK, then I will break it on Darwin, and wait for the Darwin guys.